language/German/strings.po

```diff
@@ -2112,7 +2134,7 @@ msgstr "Software-Medien"
 #: src/frontend/mame/ui/dirmenu.cpp:46
 msgctxt "path-option"
 msgid "Sound Samples"
-msgstr ""
+msgstr "Sound-Samples"
 
 #: src/frontend/mame/ui/dirmenu.cpp:47
 msgctxt "path-option"
```

Comment:
Note that here, artwork is still translated as “Illustrationen”.

Reply:
It's consistent now, since I changed the other reference to _Illustrationen_ as well.